js/apps/account-ui/maven-resources/theme/keycloak.v3/account/index.ftl

```diff
@@ -104,11 +104,7 @@
     <div id="app">
       <main class="container">
         <div class="keycloak__loading-container">
-          <span class="pf-c-spinner pf-m-xl" role="progressbar" aria-valuetext="Loading&hellip;">
-            <span class="pf-c-spinner__clipper"></span>
-            <span class="pf-c-spinner__lead-ball"></span>
-            <span class="pf-c-spinner__tail-ball"></span>
-          </span>
+          <svg class="pf-v5-c-spinner pf-m-xl" role="progressbar" aria-valuetext="Loading..." viewBox="0 0 100 100" aria-label="Contents"><circle class="pf-v5-c-spinner__path" cx="50" cy="50" r="45" fill="none"></circle></svg>
```

Comment:
This must be a bug in PatternFly, but `aria-label` is used for screen readers and should probably be set to `Loading&hellip;`. Best practices for the 'indeterminate' state should be followed here as [outlined by MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role):

> The read-only `aria-valuenow` should be provided and updated unless the value is `indeterminate`, in which case don't include the attribute.

It might be worth it to contribute this as a fix to PatternFly as well.